app/components/history_version_component.html.erb

```diff
@@ -112,6 +112,26 @@
                         <dd class="text-md p-1 break-words"><%= v %></dd>
                       </div>
                     <% end %>
+
+                    <% keys_no_longer_existent =
+                      prev_changes_value_hash.keys - changes_from_prev_version_hash.keys %>
+
+                    <% if keys_no_longer_existent.length.positive? %>
+                      <div class="col-span-3">
+                        <p class="text-red-500 dark:text-red-400"><%= t(:"components.history_version.keys_deleted") %></p>
+                      </div>
+                      <% keys_no_longer_existent.each do |key_no_longer_existent| %>
+                        <div class="col-span-1">
+                          <dt class="mb-1 text-slate-500 text-md dark:text-slate-400"><%= key_no_longer_existent %></dt>
+                        </div>
+                        <div class="col-span-1 bg-slate-100 dark:bg-slate-700 dark:text-slate-700">
+                          <dd class="text-md p-1"><%= prev_changes_value_hash[key_no_longer_existent] %></dd>
+                        </div>
+                        <div class="col-span-1 bg-slate-100 dark:bg-slate-700 dark:text-slate-700">
+                          <dd class="text-md p-1 break-words"></dd>
+                        </div>
+                      <% end %>
+                    <% end %>
```

Comment:
**nitpick:** Can you figure out where the following artifact is coming from
![image](https://github.com/phac-nml/irida-next/assets/11295750/9c4f7841-eb1c-4a83-b850-46108509d03a)

Reply:
As per our discussion tailwind needed to be started up :)

Reply:
My bad, sorry